cypress/e2e/hub/execution-environments.cy.ts

```diff
@@ -38,6 +38,8 @@ describe('Execution Environments', () => {
       cy.intercept('GET', hubAPI`/_ui/v1/execution-environments/registries/?limit=50`).as(
         'registries'
       );
+      // FIXME: this is sometimes add-.., sometimes create-..., depending on whether there are already any added
```

Comment:
(addressing this bit properly in https://github.com/ansible/ansible-ui/pull/2636)